ICSharpCode.Decompiler/TypeSystem/DecompilerTypeSystem.cs

```diff
@@ -260,19 +260,14 @@ private async Task InitializeAsync(PEFile mainModule, IAssemblyResolver assembly
 				{
 					referencedAssemblies.Add(asm);
 					var metadata = asm.Metadata;
-					foreach (var h in metadata.ExportedTypes)
```

Comment:
why would you remove support for exported types? Essentially now the code just visits all assembly references twice...